docs: updates the shell and settings pages for endless

pari-deshmukh opened the pull request:
Customizes the visual overview help page along with the accessibility, shell and settings user docs as per the user interface and also updates all the introduction page screenshots accordingly.
Also re-implements Endless specific customizations to GNOME User Docs. 

https://phabricator.endlessm.com/T25587

eos-shell-build commented:
I don't recognize the submitter as a member of @endlessm. Can somebody in @endlessm verify the patch? See https://phabricator.endlessm.com/w/software/build/ci-integration/#pull-requests for more details.

wjt commented:
@eos-shell-build ok to test

wjt commented on gnome-help/C/backup-check.page:
```diff
@@ -39,11 +39,9 @@
  and folders you transferred are actually there in the backup, you can have
  extra confidence that the process was successful.</p>
 
- <comment> This doesn't apply to Endless OS
```
No need to keep this commit and the preceeding commit, which just cancel each other out.

wjt commented on gnome-help/C/backup-how.page:
```diff
@@ -52,8 +52,7 @@
  programmer, you do not have to back up the files that get produced when you
  compile your programs. Instead, just make sure that you back up the original
  source files.</p></item>
- <item><p>Any files in the Trash folder. Your Trash folder can be found in
```
I don't think this hunk helps; in fact it removes useful information which is correct on Endless OS. Let's drop it.

pari-deshmukh replied:
Reverted in new changeset!

andrunko reviewed (changes requested):
Looking good, some minor comments below.

wjt commented on gnome-help/C/accounts-add.page:
```diff
@@ -64,7 +64,7 @@
     <item>
       <p>If you have entered your credentials correctly, you will be prompted
-      to allow GNOME access to your online account. Authorize access to
+      to allow Endless access to your online account. Authorize access to
```
It's been a while since I ran through this process but I am 90% sure that the prompt will be to allow GNOME access to your online account. We don't override the Google API key that GNOME uses.

wjt replied:
(This should be easy to test.) If it does indeed still show GNOME, no change is needed here.

pari-deshmukh replied:
I tested it out and it still shows GNOME. So reverting this back to GNOME.

wjt commented on gnome-help/C/net-firewall-on-off.page:
```diff
@@ -48,7 +48,7 @@
   <steps>
     <item>
-      <p>Go to <gui>Activities</gui> in the top left corner of the screen and
+      <p>Go to <gui>Overview</gui> in the top left corner of the screen and
```
Agreed.

pari-deshmukh commented on gnome-help/C/a11y-font-size.page:
```diff
@@ -60,7 +60,8 @@
-  <app>Tweaks</app> to make text size bigger or smaller.</p>
+  <p><gui>Large Text</gui> will scale the text by 1.2 times.</p>
+  # GNOME Tweaks not available
+  # You can use <app>Tweaks</app> to make text size bigger or smaller.
```
Removed!

pari-deshmukh commented on gnome-help/C/a11y-icon.page:
```diff
@@ -18,27 +18,27 @@
-    <desc>The accessibility menu can be found on the top bar.</desc>
-    <media its:translate="no" type="image" mime="image/png" src="figures/preferences-desktop-accessibility-symbolic.svg"/>
+    <desc>The accessibility menu can be found on the taskbar.</desc>
+    <media its:translate="no" type="image" mime="image/png" src="figures/universal-access-menu.png"/>
```
Fixed!

pari-deshmukh commented on gnome-help/C/a11y-screen-reader.page:
```diff
@@ -26,11 +26,14 @@
+  on how you installed your system, you might not have Orca installed. If not,
+  install Orca first.
   <p if:test="action:install"><link style="button" action="install:orca">Install Orca</link></p>
+  </comment>
```
Dropped the comment section here!

pari-deshmukh commented on gnome-help/C/accounts-remove.page:
```diff
@@ -36,20 +36,20 @@
 
   <note style="tip">
-    <p>Many online services provide an authorization token which GNOME stores
+    <p>Many online services provide an authorization token which Endless stores
```
Fixed!

pari-deshmukh commented on gnome-help/C/shell-exit.page:
```diff
@@ -121,7 +121,7 @@
 
   <p>To suspend your computer manually, click the system menu on the right side
-  of the top bar, expand <gui>Power Off / Log Out</gui>, and select
+  of the taskbar, expand <gui>Power Off / Log Out</gui>, and select
```
Fixed!

pari-deshmukh commented on gnome-help/C/shell-introduction.page:
```diff
@@ -69,46 +90,68 @@
-  also press the <key xref="keyboard-key-super">Super</key> key to see an
-  overview with live thumbnails of all the windows on the current workspace.</p>
+  <p if:test="platform:endless">To access your windows and applications,
+  click the Endless logo button at the bottom left corner of the screen in the window list.</p>
```
Removed!

pari-deshmukh commented on gnome-help/C/shell-introduction.page:
```diff
@@ -69,46 +90,68 @@
-  the dash to make it a favorite. Your favorite applications stay in the dash
-  even when they’re not running, so you can access them quickly.</p>
-
+  <p if:test="platform:endless">On the bottom left corner of the screen, you will find a list of
```
Makes sense! Fixed this.

pari-deshmukh commented on gnome-help/C/shell-introduction.page:
```diff
@@ -69,46 +90,68 @@
+  the windows on the current workspace.</p>
+
+  <p if:test="platform:endless">Clicking the Endless logo button allows you to toggle between
+  the Endless desktop and the running applications.</p>
```
Fixed!

pari-deshmukh commented on gnome-help/C/shell-introduction.page:
```diff
@@ -274,6 +334,14 @@ messages will be presented, such as when your battery is critically low.</p>
+    <media type="image" src="figures/shell-window-list-endless.png" width="400" if:test="!target:mobile">
+      <p>Window List</p>
+    </media>
+    <p>The window list at the bottom of the screen provides access to all your
```
Fixed!

andrunko reviewed (changes requested):
Looks mostly good but looks like some review comments got missed during this round of updates. I also left a few extra comments that I missed during initial review.

pari-deshmukh commented on gnome-help/C/a11y-icon.page:
```diff
@@ -52,8 +56,8 @@
-  keyboard focus to the top bar. A white line will appear underneath the
-  <gui>Activities</gui> button — this tells you which item on the top bar is
+  keyboard focus to the taskbar. A white line will appear underneath the
+  <gui>Overview</gui> button — this tells you which item on the taskbar is
```
Fixed!

pari-deshmukh commented on gnome-help/C/backup-thinkabout.page:
```diff
@@ -39,7 +39,7 @@
   <p>Personal files (documents, music, photos and videos)</p>
   <p its:locNote="translators: xdg dirs are localised by package xdg-user-dirs
-  and need to be translated.  You can find the correct translations for your
+  and need to be translated. You can find the correct translations for your
```
Makes sense!

pari-deshmukh commented on gnome-help/C/shell-introduction.page:
```diff
@@ -31,36 +32,56 @@
+  </if:when>
+  <if:when test="platform:endless">
+    <media type="image" src="figures/endless-desktop.png" width="600" if:test="!target:mobile">
+      <p>Endless Desktop</p>
```
Fixed!

pari-deshmukh commented on gnome-help/C/shell-introduction.page:
```diff
@@ -69,46 +90,68 @@
+  the windows on the current workspace.</p>
+
+  <p if:test="platform:endless">Clicking the Endless logo button allows you to toggle between
+  the Endless Desktop and the running applications.</p>
```
Updated!

pari-deshmukh commented on gnome-help/C/shell-introduction.page:
```diff
@@ -31,36 +32,56 @@
-  stay out of your way, minimize distractions, and help you get things done.
-  When you first log in, you will see an empty desktop and the top bar.</p>
+    <if:when test="platform:endless">
+      <p>Endless features a completely reimagined user interface designed to
```
Done!

pari-deshmukh commented on gnome-help/C/shell-introduction.page:
```diff
@@ -31,36 +32,56 @@
   </info>
 
-  <title>Visual overview of GNOME</title>
+  <title>Visual overview of Endless</title>
```
Already fixed!

pari-deshmukh commented:
@wjt @andrunko I have implemented all the changes requested and have implemented additional changes to the user docs. Also, I have rebased the branch with the latest GNOME user docs master now. Please review.

andrunko commented on gnome-help/C/display-brightness.page:
```diff
@@ -51,8 +51,8 @@
-  <gui xref="shell-introduction#systemmenu">system menu</gui> on the right side
-  of the top bar and adjust the screen brightness slider to the value you want
+  <gui xref="shell-introduction#usermenu">user menu</gui> on the right side
+  of the taskbar and adjust the screen brightness slider to the value you want
```
Also "system menu" for screen brightness.

andrunko commented on gnome-help/C/display-night-light.page:
```diff
@@ -49,9 +49,9 @@
+        <p>The <link xref="shell-introduction">taskbar</link> shows when
         <gui>Night Light</gui> is active. It can be temporarily disabled from
-        the system menu.</p>
+        the user menu.</p>
```
"system menu" also for night light.

andrunko commented on gnome-help/C/net-manual.page:
```diff
@@ -81,8 +81,8 @@
       <p>Click <gui>Apply</gui>. If you are not connected to the network, open the
-      <gui xref="shell-introduction#systemmenu">system menu</gui> from the right side of
-      the top bar and connect. Test the network
+      <gui xref="shell-introduction#usermenu">user menu</gui> from the right side of
```
"system menu"

andrunko reviewed (changes requested):
Just some small notes, looking good.

wjt commented on gnome-help/C/.yelp-tools.cfg:
```diff
@@ -0,0 +1,46 @@
+xi = http://www.w3.org/2001/XInclude
+
+[default]
+version = gnome:40
```
Will this have any visible effect in the rendered help? If so, it'll be wrong because we will be shipping GNOME 3.38. If not, then it's harmless to have this here.

We could rebase onto the commit before this one, side-stepping the problem.

pari-deshmukh replied:
This seems to be configuration very specific to yelp tools and as such doesn't show any visible effects in the rendered help docs.

sayak-sarkar commented on gnome-help/C/touchscreen-gestures.page:
```diff
@@ -47,9 +47,9 @@
+  <tr xml:id="usermenu">
     <td><p><media type="image" src="figures/right-edge-drag-down.svg" width="256" its:translate="no"/></p></td>
-    <td><p><em>Bring down the system menu</em></p>
+    <td><p><em>Bring down the user menu</em></p>
```
I'm not sure how to verify this, and as such skipping this changeset for now.
@wjt Do you have any idea about Endless OS's Touch-screen gestures?

pari-deshmukh replied:
Not sure how this would look like on a touchscreen device. Maybe @wjt has some idea?

andrunko commented:
Thanks @pari-deshmukh, landed these changes + a few fixup commits on top.
